src/components/footer/Footer.component.tsx

```diff
@@ -0,0 +1,23 @@
+import { IconsComponent } from '@/components/icons/Icons.component.tsx';
+
+import footerStyles from './footer.module.css';
+
+export const FooterComponent = () => (
+    <div className={footerStyles.container}>
+        <div className={footerStyles.divider}>
+            <svg xmlns="http://www.w3.org/2000/svg" width="1148" height="2" viewBox="0 0 1148 2" fill="none">
```

Comment:
Using CSS to create simple elements like dividers is often a better approach for better consistency and easier maintenance.